README.md

```diff
@@ -218,6 +219,43 @@ void interactive_console(dmlog_ctx_t ctx) {
     }
 }
 
+void interactive_console_with_firmware_echo(dmlog_ctx_t ctx) {
+    // Request input with echo disabled - firmware will handle echo
+    // This allows firmware to implement command completion, special keys, etc.
+    dmlog_input_request(ctx, DMLOG_INPUT_FLAG_ECHO_OFF);
+    
+    // Wait for input to become available
+    while (!dmlog_input_available(ctx)) {
+        // Could do other work here or yield to other tasks
+    }
+    
+    // Read input character by character and echo it
+    char c;
+    while ((c = dmlog_input_getc(ctx)) != '\0') {
```

Comment:
Przecież ten przykład bajt po bajcie nie będzie działać bo nie ustawiłeś tej flagi byte więc dopiero jak użytkownik wciśnie enter to się to wyśle